internal/docstore/firedocstore/codec.go

```diff
@@ -44,12 +44,12 @@ import (
 // The Firestore proto definition for Value is a oneof of various types,
```

Comment:
The comment should also be updated.

Reply:
I'm sorry, I don't understand what you mean.

Reply:
The function's return type is changed from Value to Document, the comment should say Document instead of "Value"

Reply:
I think I understand. PTAL.